sklearn/ensemble/bagging.py

```diff
@@ -541,6 +541,37 @@ class BaggingClassifier(ClassifierMixin, BaseBagging):
         `oob_decision_function_` might contain NaN. This attribute exists
         only when ``oob_score`` is True.
 
+
+    Examples
+    --------
+    >>> from sklearn.dummy import DummyClassifier
+    >>> from sklearn.ensemble import BaggingClassifier
+    >>> from sklearn.datasets import make_classification
+    >>> X, y = make_classification(n_samples=1000, n_features=4,
+    ...                            n_informative=2, n_redundant=0,
+    ...                            random_state=0, shuffle=False)
+    >>> clf = BaggingClassifier(n_estimators=100, random_state=0).fit(X, y)
+    >>> # BaggingClassifier(base_estimator=None, bootstrap=True,
+    >>> #                   bootstrap_features=False, max_features=1.0,
+    >>> #                   max_samples=1.0, n_estimators=100,
+    >>> #                   n_jobs=None, oob_score=False,
+    >>> #                   random_state=0, verbose=0,
+    >>> #                   warm_start=False)
+    >>> print(clf.predict([[0, 0, 0, 0]]))
+    [1]
+    >>> clf = BaggingClassifier(base_estimator=DummyClassifier(),
```

Comment:
I would use something smarter than _Dummy_ ;) Since these code snippets get copy/pasted, using a more sensible option wouldn't be a bad idea.

Reply:
Done